lcm/dbg.h

```diff
@@ -112,32 +90,21 @@ typedef struct dbg_mode_color {
     const char *color;
 } dbg_mode_color_t;
 
+static dbg_mode_color_t dbg_colortab[] = {DBG_COLORTAB};
 
-static dbg_mode_color_t dbg_colortab[] = {
-    DBG_COLORTAB
-};
-
-static dbg_mode_t dbg_nametab[] = {
-    DBG_NAMETAB
-};
+static dbg_mode_t dbg_nametab[] = {DBG_NAMETAB};
 
-static inline 
-const char* DCOLOR(unsigned long long d_mode)
-{
+static inline const char *DCOLOR(unsigned long long d_mode) {
```

Comment:
This represents a change to a different brace style. Do we *really* want that?

Reply:
You're right. I'll change the brace style. For some reason I thought there was a mixture of both, but it looks like mostly functions have the brace on the next line.